Devscord.DiscordFramework/Services/EmbedMessagesService.cs

```diff
@@ -6,7 +6,10 @@ namespace Devscord.DiscordFramework.Services
 {
     public class EmbedMessagesService
     {
-        public Embed Generate(string title, string description, IEnumerable<KeyValuePair<string, string>> values)
+        internal int FooterLength => FOOTER_TEXT.Length;
```

Comment:
a czemu to jest jako internal?

Reply:
tak trochę z rozpędu dałem internal, bo we wszystkich pozostałych metodach dałem
chociaż ta jedna informacja i tak nie ma większego sensu na zewnątrz, więc może to i dobrze że internal 🤔